<launch>

    <include file="$(find zed_cpu)/launch/zed_cpu.launch"/>

    <arg name="size" default="7x7"/>
    <arg name="square" default="0.05"/>
    <arg name="no-service-check" default="true"/>

    <node pkg="camera_calibration" type="cameracalibrator.py" name="camera_calibration" output="screen" args="">
        <remap from="right" to="/camera/right/image_raw"/>
        <remap from="left" to="/camera/left/image_raw"/>
    </node>

</launch>
